services/exposurelog/values-summit.yaml

```diff
@@ -0,0 +1,21 @@
+exposurelog:
+
+  nfs_path_1: /readonly/comcam
+  nfs_server_1: comcam-arctl01.cp.lsst.org
+  butler_uri_1: /volume_1  # a guess
+
+  ingress:
+    enabled: true
+    annotations: {}
+    #   kubernetes.io/ingress.class: nginx
+    #   kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: lsst-nts-k8s.ncsa.illinois.edu
```

Comment:
That should be summit-lsp.lsst.codes

Reply:
Thank you for catching that and for the corrected value.